packages/dates_calc/dates_calc.0.0.7/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A date calculation library"
+description: """
+A date calculation library, with exact operators to add a given
+ number of days to a date, and approximate operators to add months or
+ years."""
+maintainer: ["Raphaël Monat <raphael.monat@lip6.fr>"]
+authors: ["Aymeric Fromherz" "Denis Merigoux" "Raphaël Monat" "Louis Gesbert"]
+license: "Apache-2.0"
+homepage: "https://github.com/CatalaLang/dates-calc"
+bug-reports: "https://github.com/CatalaLang/dates-calc/issues"
+depends: [
+  "dune" {>= "3.11"}
+  "ocaml" {>= "4.11.0"}
+  "alcotest" {with-test & >= "1.5.0"}
+  "qcheck" {with-test & >= "0.15"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
#    "@runtest" {with-test} # require python 3.12 to run
```

Reply:
To disable them in your repo, what you could do is

- add a dummy `dates_calc_python_test.opam`
_ add `(package dates_calc_python_test)` to [`test/dune](https://github.com/CatalaLang/dates-calc/blob/9125d78833e092b5be009f23b7013a753018466a/test/dune#L7)

Then on release you only release `dates_calc`